docs/reference/resource-limits-fees.mdx

```diff
@@ -13,31 +13,13 @@ Resource limitations and fees only apply to smart contract transactions. Read mo
 
 Soroban introduces smart contracts, marking the most transformative upgrade to the Stellar network to date. To maintain network performance, the ecosystem has decided to upgrade the network in a phased approach where network capacity is increased gradually over time. See [this blog post](https://stellar.org/blog/developers/the-stellar-network-s-phased-rollout-of-smart-contracts-the-road-to-mainnet) for more details and [Discord](https://discord.com/invite/zVYdY3ktTn) for current status.
 
-| Network Setting | Phase 0 | Phase 1 (current) |
-| :-- | :-- | :-- |
-| Soroban Txn per ledger | 1 | 100 |
-| Max CPU Instructions per Txn | 2.5 million (2M instructions for max Wasm) | 100 million (VM instantiation consumes up to 23M for a 64KB Wasm, and 46M for multiple Wasms of 130KB) |
-| Memory limit per Txn | 2 MB | 40 MB |
-| Ledger entry size (including Wasm entries) per Txn | 2 KB | 64 KB |
-| Read/Write Ledger entries per Txn | 3 read; 2 write | 40 read; 25 write |
-| Read/Write bytes per Txn | 3.2 KB read; 3.2 KB write | 130 KB read; 65 KB write |
-| Transaction size | 10 KB | 70 KB |
-| Persistent entry minimal/initial lifetime | 4,096 ledgers (~5.68 hours) | 120 days |
-| Temporary entry minimal/initial lifetime | 16 ledgers | 1 day |
-| Max ledger entry expiration bump | 61 days | 6 months |
-| Events+return value size bytes | 200 B | 8 KB |
-
 ## Resource Fees
 
 Note that write fees grow linearly from empty ledger to ledger "target size", and then grow linearly, but with a 1000x factor after exceeding the target. This is to bound the ledger size growth.
```

Comment:
This may just be me, but the first sentence doesn't read quite right. We say that the "...write fees grow linearly from empty ledger to ledger "target size", and then grow linearly..." so we say that write fees grow linearly twice. Is that purposeful?

Reply:
The distinction is in the next half of the sentence: "... but with a 1000x factor after exceeding the target." As it's written, it is quite confusing, but I'm not sure a better way to say it.

Essentially, the write fee can be affected by the current size of the blockchain (that's what `ledger "target size"` is referring to [i'm pretty sure...]):

1. while `size_of_blockchain >= "target size"` ... write fee = `fee_that_grows_linearly`
2. while `size_of_blockchain > "target size"` ... write fee = `fee_that_grows_linearly * 1000`

@dmkozh Could probably say definitively if I understand it correctly.

Reply:
Your understanding is correct. Also please feel free to come up with a better term than 'target ledger size'. Maybe we should use something like 'ledger growth threshold' and clarify that the threshold will grow over time and it's introduced only to prevent explosive state growth.